project/Build.scala

```diff
@@ -31,7 +31,7 @@ object ScaldingBuild extends Build {
   val hadoopLzoVersion = "0.4.19"
   val hadoopVersion = "2.5.0"
   val hbaseVersion = "0.94.10"
-  val hravenVersion = "0.9.16"
+  val hravenVersion = "0.9.17.t05"
```

Comment:
Yep it's on maven central and sonatype 
https://oss.sonatype.org/content/repositories/releases/com/twitter/hraven/hraven-core/